### DISCUSSION TITLE
How can i pass Model to wire-elements/modal as parameter?

### DISCUSSION THREAD

**Discussion by rosolovsky** [Power-Components/livewire-powergrid#589] [Q&A]
How can i pass Model to wire-elements/modal as parameter?

```
Button::add('new-modal')
       ->caption('Edit')
       ->openModal('edit-category-modal', ['model' => 'model'])
```

**Comment by dansysanalyst** [comment]
Hi @rosolovsky,

You can see an example here:

https://github.com/Power-Components/powergrid-demo/blob/main/app/Http/Livewire/DishesTable.php

https://github.com/Power-Components/powergrid-demo/blob/main/app/Http/Livewire/DeleteDish.php
